cpp/include/ray/api.h

```diff
@@ -189,5 +220,30 @@ ActorCreator<F> Ray::Actor(F create_func) {
   return CreateActorInternal<F>(create_func);
 }
 
+template <typename T>
+boost::optional<ActorHandle<T>> Ray::GetActorInternal(bool global,
+                                                      const std::string &actor_name) {
+  if (actor_name.empty()) {
+    return {};
+  }
+
+  auto actor_id = ray::internal::RayRuntime()->GetActorId(global, actor_name);
+  if (actor_id.empty()) {
+    return {};
+  }
+
+  return ActorHandle<T>(actor_id);
+}
+
+template <typename T>
+boost::optional<ActorHandle<T>> Ray::GetActor(const std::string &actor_name) {
```

Comment:
It seems that we don't support creating a non-global named actor yet. And we also don't have a test case for `Ray::GetActor`. Is it intentional?

Reply:
Creating a non-global named actor is not supported now, i will remove the api in this pr.